app/scripts/metamask-controller.js

```diff
@@ -711,6 +711,7 @@ export default class MetamaskController extends EventEmitter {
         'MultichainNetworkController:networkDidChange',
       ],
       allowedActions: [
+        'KeyringController:getState',
```

Comment:
Now required since `@metamask/accounts-controller@27.0.0`.

Reply:
@ccharly on the pull request I see we have removed `KeyringController:getAccounts` and `KeyringController:getKeyringForAccount` do you think we can remove them ?

Reply:
Nice catch @cryptodev-2s! I think you're right, we can also remove those, let me do that, thanks!